benchmark/benchmain/main.go

```diff
@@ -389,51 +411,59 @@ func makeFuncStream(bf stats.Features) (rpcCallFunc, rpcCleanupFunc) {
 		if bf.RespPayloadCurve != nil {
 			respSizeBytes = bf.RespPayloadCurve.ChooseRandom()
 		}
-		streamCaller(streams[pos], reqSizeBytes, respSizeBytes)
+		streamCaller(streams[conn][pos], reqSizeBytes, respSizeBytes)
 	}, cleanup
 }
 
 func makeFuncUnconstrainedStreamPreloaded(bf stats.Features) (rpcSendFunc, rpcRecvFunc, rpcCleanupFunc) {
 	streams, req, cleanup := setupUnconstrainedStream(bf)
 
-	preparedMsg := make([]*grpc.PreparedMsg, len(streams))
-	for i, stream := range streams {
-		preparedMsg[i] = &grpc.PreparedMsg{}
-		err := preparedMsg[i].Encode(stream, req)
-		if err != nil {
-			logger.Fatalf("%v.Encode(%v, %v) = %v", preparedMsg[i], req, stream, err)
+	preparedMsg := make([][]*grpc.PreparedMsg, len(streams))
+	for cn, connStreams := range streams {
+		preparedMsg[cn] = make([]*grpc.PreparedMsg, len(connStreams))
+		for i, stream := range connStreams {
+			preparedMsg[cn][i] = &grpc.PreparedMsg{}
+			err := preparedMsg[cn][i].Encode(stream, req)
+			if err != nil {
+				logger.Fatalf("%v.Encode(%v, %v) = %v", preparedMsg[i], req, stream, err)
+			}
 		}
 	}
 
-	return func(pos int) {
-			streams[pos].SendMsg(preparedMsg[pos])
-		}, func(pos int) {
-			streams[pos].Recv()
+	return func(conn, pos int) {
+			streams[conn][pos].SendMsg(preparedMsg[conn][pos])
+		}, func(conn, pos int) {
+			streams[conn][pos].Recv()
 		}, cleanup
 }
 
 func makeFuncUnconstrainedStream(bf stats.Features) (rpcSendFunc, rpcRecvFunc, rpcCleanupFunc) {
 	streams, req, cleanup := setupUnconstrainedStream(bf)
 
-	return func(pos int) {
-			streams[pos].Send(req)
-		}, func(pos int) {
-			streams[pos].Recv()
+	return func(conn, pos int) {
```

Comment:
Nit: I see conn and cn throughout this PR for the first dimension of conn index (whether a function parameter or an index declaration in a for iteration). I also see pos and i for the steam index within the conn. Please make consistent across PR.

Reply:
Done, used cn and pos thought the PR.